chapter_convolutional-modern/vgg_vn.md

```diff
@@ -212,7 +215,10 @@ d2l.train_ch6(net, train_iter, test_iter, num_epochs, lr)
 4. Refer to Table 1 in :cite:`Simonyan.Zisserman.2014` to construct other common models, such as VGG-16 or VGG-19.
 -->
 
-*dịch đoạn phía trên*
+1. Khi in ra kích thước của các lớp, chúng ta chỉ thấy 8 kết quả chứ không phải 11. Thông tin về 3 lớp còn lại nằm ở đâu?
+2. So với AlexNet, VGG chậm hơn đáng kể về mặt tính toán, và nó cùng đòi hỏi nhiều bộ nhớ GPU hơn. Hãy phân tích lý do cho hiện tượng này?
```

Comment:
2. So với AlexNet, VGG chậm hơn đáng kể về mặt tính toán, và nó `cũng` đòi hỏi nhiều bộ nhớ GPU hơn. Hãy phân tích lý do cho hiện tượng này?